### PR TITLE
ci: use Dockerfile for Jenkins CI builds

### PR DESCRIPTION
**Pull request by jakubgs**
This way we have control over our system dependencies.

Resolves: https://github.com/status-im/help.status.im/issues/184

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -38,11 +42,9 @@ pipeline {
 
     stage('Build') {
       steps {
-        sh "python -m mkdocs build -f config/en/mkdocs.yml"
-        sh 'python -m mkdocs build -f config/style-guide/mkdocs.yml'
+        sh 'python3 -m mkdocs build -f config/en/mkdocs.yml'
         /* Temporary solution for lack of start page. */
         sh 'cp overrides/static/* generated/'
-        echo "${GIT_BRANCH}"
```
Comment (jorge-campo) [inline Jenkinsfile:45]: Hey @jakubgs, where is now this `GIT_BRANCH` parameter?

Comment (jakubgs) [reply Jenkinsfile:45]: >where is now this GIT_BRANCH parameter?

What do you mean? That was just a useless `echo` leftover from some debugging.

Comment (jakubgs) [reply Jenkinsfile:45]: Also, check now, it was cached.

**Comment by jorge-campo** [comment]
Superseded by https://github.com/status-im/help.status.im/issues/195.

**Comment by jakubgs** [comment]
What do you mean by "superseded"? "Related", sure, but it would be "superseded" if that PR had a different and newer implementation of the build setup, not a fix.

**Pull request merged**